king_phisher/server/database/models.py

```diff
@@ -397,6 +398,95 @@ class Message(Base):
 	credentials = sqlalchemy.orm.relationship('Credential', backref='message', cascade='all, delete-orphan')
 	visits = sqlalchemy.orm.relationship('Visit', backref='message', cascade='all, delete-orphan')
 
+@register_table
+class ModuleQuestion(Base):
+	__repr_attributes__ = ('question',)
+	__tablename__ = 'module_questions'
+	id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+	question = sqlalchemy.Column(sqlalchemy.String, nullable=False)
+	question_type = sqlalchemy.Column(sqlalchemy.String, nullable=False)
+	hint = sqlalchemy.Column(sqlalchemy.String)
+	url_reference = sqlalchemy.Column(sqlalchemy.String)
+	possible_answer_group_id = sqlalchemy.Column(sqlalchemy.Integer, nullable=False)
+	question_answer_id = sqlalchemy.Column(sqlalchemy.Integer, sqlalchemy.ForeignKey('question_answers.id'))
+	# relationships
+	submission_answer_link = sqlalchemy.orm.relationship('SubmissionAnswerLink', backref='module_questions', cascade='all, delete-orphan')
+
+@register_table
+class Module(TagMixIn, Base):
+	__tablename__ = 'modules'
+	created = sqlalchemy.Column(sqlalchemy.DateTime, nullable=False)
```

Comment:
All of these new `create` fields you're adding should use `default=current_timestamp` and then drop the nullable constraint since you'll have a default value.